pkg/daemon/ceph/client/pool.go

```diff
@@ -92,7 +92,7 @@ func ListPoolSummaries(context *clusterd.Context, clusterInfo *ClusterInfo) ([]C
 	args := []string{"osd", "lspools"}
 	output, err := NewCephCommand(context, clusterInfo, args).Run()
 	if err != nil {
-		return nil, errors.Wrap(err, "failed to list pools")
+		return nil, errors.Wrapf(err, "failed to list pools: \n%s", output)
```

Comment:
Was this helpful for troubleshooting the tests, or why is it needed? Adding a newline in the `errors.Wrapf()` may not be formatted well.